### ISSUE TITLE
Windows compat: duplicate icons column

### ISSUE THREAD

**Issue by fleimgruber** [alexluigit/dirvish#16]
Windows 10, Emacs 27.2.50, dirvish-20220316.720

Steps to reproduce:

1. M-x dirvish
2. M-x dirvish-setup-menu
3. i
4. i

After step 1:

![2022-03-16 12_27_53-Window](https://user-images.githubusercontent.com/7400038/158581161-56219d94-9e30-4ce8-9513-4f57513c139d.png)

After step 3:

![2022-03-16 12_29_11-Window_dirvish_icons](https://user-images.githubusercontent.com/7400038/158581338-ce8dd209-8efd-49cf-9dad-f04fd585e23b.png)

After step 4:

![2022-03-16 12_30_12-Window_no_dirvish_icons](https://user-images.githubusercontent.com/7400038/158581363-addee0e9-e7a8-4f4e-b31d-f15dffff9343.png)

**Comment by fleimgruber** [comment]
For the "neotree" pane to the left the icons are replicated 3 times

![image](https://user-images.githubusercontent.com/7400038/158582883-2845dc16-05fb-4817-a000-8fe0bda6e537.png)

**Comment by alexluigit** [comment]
What is the value of your `dirvish-attributes`?
If you set `dirvish-attributes` to nil, how does it look like?

**Comment by fleimgruber** [comment]
dirvish-attributes: nil
dirvish-icon-backend: all-the-icons

`emacs -q ...` leads to similar results as you showed (i.e. correct behavior).

Any ideas which user config could mess up the display as shown in initial post?

**Comment by fleimgruber** [comment]
Another thing that might represent a lead, initially the icons are duplicated twice in the neotree pane:

![image](https://user-images.githubusercontent.com/7400038/158598729-2f16a259-05d4-4846-891a-42b13cfe7cb2.png)

**Comment by alexluigit** [comment]
If your `dirvish-attributes` evals to nil, then these icons are not rendered by dirvish. I don't use spacemacs myself so I'm not sure where these icons came from. Do you have treemacs installed? Maybe it added some hooks in dired-mode-hook.

**Comment by fleimgruber** [comment]
Good point, treemacs is installed, but I did not find a way of testing without it yet. Will check back later.

**Comment by alexluigit** [comment]
This should work. But a better solution would be inhibit loading `treemacs-icons-dired` in the first place.

```emacs-lisp
(remove-hook 'dired-mode-hook 'treemacs-icons-dired-mode)
(remove-hook 'dired-after-readin-hook 'treemacs-icons-dired--display)
```

**Comment by alexluigit** [comment]
>Might be good to have a Spacemacs compat section? I could come up with a draft for that.

I agree with the proposal and thanks for your help :)
